cmake/QoreMacros.cmake

```diff
@@ -287,15 +287,17 @@ MACRO (QORE_USER_MODULE _module_file _mod_deps)
         endforeach(i)
 
         # prepare QDX arguments
-        set(QDX_DOXYFILE_ARGS -M=${CMAKE_SOURCE_DIR}/${_module_file}:${CMAKE_BINARY_DIR}/doxygen/qlib/${f}.qm.dox.h ${MOD_DEPS} ${CMAKE_SOURCE_DIR}/doxygen/qlib/Doxyfile.tmpl ${MOD_DOXYFILE})
-        set(QDX_QMDOXH_ARGS ${CMAKE_SOURCE_DIR}/${_module_file} ${CMAKE_BINARY_DIR}/doxygen/qlib/${f}.qm.dox.h)
+        #set(QORE_QMOD_FNAME ${f}) # used for configure_file line below   done by qdx other way for each module
+        configure_file(${CMAKE_SOURCE_DIR}/doxygen/qlib/Doxyfile.in ${CMAKE_BINARY_DIR}/doxygen/Doxyfile.qlib.in @ONLY)
+        set(QDX_DOXYFILE_ARGS -M=${CMAKE_SOURCE_DIR}/${_module_file}:${CMAKE_BINARY_DIR}/doxygen/qlib/${f}.qm ${MOD_DEPS} ${CMAKE_BINARY_DIR}/doxygen/Doxyfile.qlib.in ${MOD_DOXYFILE})
```

Comment:
this is really long line, not easily readable, can it be splitted?

Reply:
No idea how cmake treats with CRLFs assigned to string used as cmdline parameter